### PR TITLE
Update readme

### LINKED ISSUE #4: Add more documentation to Readme
- [x] Add some quickstart examples

### PR DESCRIPTION
**Pull request by svenvanderburg**
* Update README with a bit more introduction to the project, a quick start section, and a bit on contributing
* Add tutorial notebook that is a more extensive working example of the quick start section
* Add CONTRIBUTING.md

Closes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,27 +7,84 @@
...
+## Setup
+### Requirements
 
 Python 3.7 or higher
 
-## Prepare environmnent
+### Installation
+Simply install using pip: `pip install ms2deepscore`
+
+###Prepare environment
```
Comment (florian-huber) [inline README.md:28]:
```suggestion
### Prepare environment
```

File: README.md [hunk 2]
```diff
@@ -7,27 +7,84 @@
...
+### Predict similarity scores
+Calculate similariteis for a pair of spectra
+```python
+from ms2deepscore import MS2DeepScore
+similarity_measure = MS2DeepScore(model)
+score = similarity_measure.pair(spectrums[0], spectrums[1])
+```
+
+## Contributing
+We welcome contributions to the development of ms2deepscore! Have a look at the [contribution guidelines](ttps://github.com/matchms/ms2deepscore/blob/main/CONTRIBUTING.md).
```
Comment (florian-huber) [inline README.md:90]:
```suggestion
We welcome contributions to the development of ms2deepscore! Have a look at the [contribution guidelines](https://github.com/matchms/ms2deepscore/blob/main/CONTRIBUTING.md).
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+## You have a question
+
+1. use the search functionality [here](https://github.com/iomega/spec2vec/issues) to see if someone already filed the same issue;
+1. if your issue search did not yield any relevant results, make a new issue;
+1. apply the "Question" label; apply other labels when relevant.
+
+## You think you may have found a bug
+
+1. use the search functionality [here](https://github.com/iomega/spec2vec/issues) to see if someone already filed the same issue;
```
Comment (florian-huber) [inline CONTRIBUTING.md:22]:
```suggestion
1. use the search functionality [here](https://github.com/matchms/ms2deepscore/issues) to see if someone already filed the same issue;
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+## You want to make some kind of change to the code base
+
+1. (**important**) announce your plan to the rest of the community *before you start working*. This announcement should be in the form of a (new) issue;
+1. (**important**) wait until some kind of consensus is reached about your idea being a good idea;
+1. if needed, fork the repository to your own Github profile and create your own feature branch off of the latest master commit. While working on your feature branch, make sure to stay up to date with the master branch by pulling in changes, possibly from the 'upstream' repository (follow the instructions [here](https://help.github.com/articles/configuring-a-remote-for-a-fork/) and [here](https://help.github.com/articles/syncing-a-fork/));
+1. make sure the existing tests still work by running ``python setup.py test``;
+1. add your own tests (if necessary);
+1. update or expand the documentation;
+1. update the `CHANGELOG.md` file with change;
+1. [push](http://rogerdudler.github.io/git-guide/>) your feature branch to (your fork of) the spec2vec repository on GitHub;
```
Comment (florian-huber) [inline CONTRIBUTING.md:38]:
```suggestion
1. [push](http://rogerdudler.github.io/git-guide/>) your feature branch to (your fork of) the ms2deepscore repository on GitHub;
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+In case you feel like you've made a valuable contribution, but you don't know how to write or run tests for it, or how to generate the documentation: don't let this discourage you from making the pull request; we can help you! Just go ahead and submit the pull request, but keep in mind that you might be asked to append additional commits to your pull request.
+
+## You want to make a new release of the code base
+
+To create release you need write permission on the repository.
+
+1. Check author list in `citation.cff` and `.zenodo.json` files
+1. Bump the version using `bump2version <major|minor|patch>`. For example, `bump2version major` will increase major version numbers everywhere its needed (code, meta, etc.) in the repo.
+1. Update the `CHANGELOG.md` to include changes made
+1. Goto [GitHub release page](https://github.com/iomega/spec2vec/releases)
```
Comment (florian-huber) [inline CONTRIBUTING.md:50]:
```suggestion
1. Goto [GitHub release page](https://github.com/matchms/ms2deepscore/releases)
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+1. Press draft a new release button
+1. Fill version, title and description field
+1. Press the Publish Release button
+
+A GitHub action will run which will publish the new version to [anaconda](https://anaconda.org/nlesc/spec2vec).
```
Comment (florian-huber) [inline CONTRIBUTING.md:55]:
```suggestion
A GitHub action will run which will publish the new version to [pypi](https://pypi.org/project/ms2deepscore).
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+Also a Zenodo entry will be made for the release with its own DOI.
```
Comment (florian-huber) [inline CONTRIBUTING.md:56]: This is not implemented yet I believe. Could become a new issue

**Review by florian-huber — CHANGES_REQUESTED** [review]
Thanks a lot for adding this Sven!
I added some small corrections, but otherwise looks great!

**Comment by svenvanderburg** [comment]
Nice catches! (I will pay a bit more attention when I copy-paste next time :) )

**Pull request merged**